yo/main.py

```diff
@@ -1508,6 +1508,37 @@ def run_for_instance(self, inst: YoInstance) -> None:
             )
 
 
+class CopyIdCmd(SingleInstanceCommand):
+    name = "copy-id"
+    description = "copy an SSH public key onto an instance using ssh-copy-id"
+
+    def add_args(self, parser: argparse.ArgumentParser) -> None:
+        parser.add_argument("instance_name", type=str, help="Name of the instance")
+        parser.add_argument("-i", "--identity-file", type=str, required=True,
+                            help="Specify path to the public key file")
```

Comment:
`SingleInstanceCommand` already has the instance name provided as an argument. You simply need to ensure that you call the super-class `add_args()`:

```suggestion
    def add_args(self, parser: argparse.ArgumentParser) -> None:
        super().add_args(parser)
        parser.add_argument("-i", "--identity-file", type=str, required=True,
                            help="Specify path to the public key file")
```

Reply:
checked and done.